File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.java
class Solution {
    public int longestConsecutive(int[] nums) {
        HashSet<Integer> set = new HashSet<>();
        int count=0, maxCount=0;;
        for(int i : nums){
            set.add(i);
        }

        for(int i : nums){
            count = 0;
            while(set.contains(i)){
                count++;
                i -= 1;
            }
            maxCount = Math.max(maxCount, count);
        }
        return maxCount;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        curr = 0
        currMax = nums[0]

        for i in nums:
            if curr < 0:
                curr = 0
            curr += i
            currMax = max(curr, currMax)
        

        return currMax